System/Posix/Env/ByteString.hsc

```diff
@@ -53,8 +53,8 @@ getEnv name = do
      else return Nothing
 
 -- |'getEnvDefault' is a wrapper around 'getEnv' where the
--- programmer can specify a fallback if the variable is not found
--- in the environment.
+-- programmer can specify a fallback as the second argument, which will be
+-- used if the variable is not found in the environment.
```

Comment:
Would it perhaps be better to instead (or also) add haddock comments for each of the parameters in the signature? Perhaps in that case leaving the original description unchanged.  The first argument can then be clearly indicated as the name of the environment variable, and the second as the fallback value.

Reply:
As the original author is no longer with us I think both is fine.